/**:
  ros__parameters:
    sub_topic_name: "/relief_detector/bar/normalized"
    stopping_lifecycle_name: "/to/your_lifecycle"
    relief_threshold: 100.0
    saving_raw_data_directory: "/home/yazawa/microros_ws/src/relief_stopper/logs/raw"
    saving_stopping_data_directory: "/home/yazawa/microros_ws/src/relief_stopper/logs/normalized"

